### PR TITLE
Janitor Buff

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>
Janitors NEVER received or at least barely received any reward when Lonestar won at the colony games AT ALL.
I want to make this PR to address and remedy the issue that the janitors barely have the tools for their job.

add: New job titles to the janitor, Maintenance Technician, and Lonestar Caretaker

rebalance: Gives janitors a buff of 10 health due to their hazardous job.

rebalance: Their stats from ROB 10 and BIO 10 to ROB 10 BIO 15 TGH 10 MEC 15 COG 5 and VIG 5
Reason is: They rather need some MEC stats to deal with... well.. literally everything their job entails. 15 should suffice? Maybe less? Butchering Corpses +5 more BIO so it can aid them.

rebalance: Their keys are now somewhat more usable, less time, and are less bulky

map/balance: Gives Janitors 2 tool belts at their cleaning shack, two grizzl with two boxes each of slugs and buckshot, two cogs with four cells (To two janitors use), a power shovel to clean burrows, welding tools, and a welding tank/water tank, a hazard vest so they can have more protection while dealing with infestations and two scavager helmets that was kindly given from the prospectors.

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by Guidesu** [comment]
Old:
image

New:
image


To people who don't know how to use the bots

**Comment by Rat-Time** [comment]
Janitor...armory?

File: code/game/objects/items/weapons/cards_ids.dm [hunk 1]
```diff
@@ -419,7 +419,7 @@ var/const/NO_EMAG_ACT = -50
 /obj/item/keys/janitor
 	name = "janitor keys"
 	desc = "A set of keys to open any door, can't open \"deadbolts\"."
-	w_class = ITEM_SIZE_BULKY //No hiding this
+	w_class = ITEM_SIZE_NORMAL // Why it was that big?
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/cards_ids.dm:422]:
```suggestion
	w_class = ITEM_SIZE_BULKY //No hiding this
```

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -1022,7 +1022,7 @@ There are 9 wires.
 				playsound(loc, 'sound/items/keychainrattle.ogg', 30, 1, -2)
 			else
 				playsound(loc, 'sound/items/keychainrattle.ogg', 700, 1, -2)
-			if(do_after(user, 300, src)) //in ms so half a min of sitting their trying
+			if(do_after(user, 150, src)) //buffs janitor key thingy.
```
Comment (Trilbyspaceclone) [inline code/game/machinery/doors/airlock.dm:1025]:
```suggestion
			if(do_after(user, 300, src)) //in ms so half a min of sitting their trying
```

File: code/game/jobs/job/civilian.dm [hunk 3]
```diff
@@ -170,29 +170,33 @@
...
 	software_on_spawn = list(/datum/computer_file/program/camera_monitor)
 	description = "The lowly Janitor, well paid but rarely respected, nevertheless provides a valuable service.<br>\
 	You will trundle around the colony, armed with your trusty janicart full of cleaning supplies, disposing of trash and pests.<br>\
-	Some minor maintenance also falls within your duties. Fix lights, correct broken vendors, replace floor tiles - ease the guild burden.<br>\
+	Some maintenance also falls within your duties. Fix lights, correct broken vendors, replace floor tiles - ease the guild burden.<br>\
 	Although your status is miserable, your knowledge of maintenance may lead you to some valuable salvage for some extra cash."
 
 	duties = "Clean blood, dirt, rubble and messes. Don't clean up crime scenes!<br>\
-		Conduct minor repairs and maintenance when guild adepts aren't available.<br>\
+		Conduct minor repairs and maintenance when guild adepts aren't available, aswell take care of te bar/kitchen/garden if nobody is there.<br>\
```
Comment (Trilbyspaceclone) [inline code/game/jobs/job/civilian.dm:199]:
```suggestion
		Conduct minor repairs and maintenance when guild adepts aren't available.<br>\
```

File: code/game/machinery/doors/airlock.dm (hunk 2 above)
Comment (Guidesu) [reply code/game/machinery/doors/airlock.dm:1025]: why it can't be faster?

**Comment by cdb-is-not-good** [comment]
kinda hate this. the janitor doesn't need an armory.

**Comment by Dissentbay** [comment]
give the guy a gun? sure
armory? nuh uh

**Comment by benj8560** [comment]
Cog as ''cheap'' as its meant to be is a pretty insane gun for a non-combat role. Their now better armed then miners with this. I'd touch it down to 2 grizzlys no cog and only buckshot. Same as a miner gets. Gives incentive to get better ammo while not being a useless weapon. 

Powered shovels proly a little much maybe switch for drills and I see powered hammers....somewhere in the code but not in the image. They can use those wherever they might be for rapidly dealing with trash piles by relocating them to guild. Speaking of the hammers could fully remove the guns. Power hammer is a pretty sick weapon of it's own. 

My suggestions for getting this merged without any rioting

**Pull request closed without merging**